packages/dev-frontend/src/strings.ts

```diff
@@ -1,4 +1,4 @@
-export const COIN = "LUSD";
+export const COIN = "thUSD";
 export const GT = "LQTY";
 export const LP = "UNI LP";
-// export const LP = "ETH/LUSD LP";
+export const ERC20 = "WETH";
```

Comment:
worth considering how this would work if we end up supporting TBTC and WETH as collateral